src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java

```diff
@@ -46,6 +56,36 @@ public static BlockingSphereClient createClient(@Nonnull final SphereClientConfi
     }
 
     /**
+     * Return a {@link SphereClient} instance which will retry its request after it receives a server error from CTP.
+     * A given {@code sphereClient} is wrapped by {@link RetrySphereClientDecorator} with proper rules initialized, and
+     * then returned.
+     *
+     * @param sphereClient a delegate that will execute requests
+     * @return {@link SphereClient} that will retry its requests on 5xx errors
+     */
+    @Nonnull
+    public static SphereClient getRetryOn5xxClient(@Nonnull final SphereClient sphereClient) {
+        return RetrySphereClientDecorator.of(sphereClient, singletonList(RetryRule.of(
+            ofMatchingErrors(ServerErrorException.class),
+            ofScheduledRetry(RETRY_ON_5XX_ATTEMPTS_LIMIT, ClientConfigurationUtils::retryOn5xxDurationCalculator))));
+    }
+
+    /**
+     * Given a {@link RetryContext} it returns duration for a current attempt. The duration is calculated from a
+     * following equation: <pre>duration = 2 ^ (((retryAttemptNumber - 1) / 3) + 1) [seconds]</pre>
```

Comment:
retry strategy: https://github.com/sphereio/sphere-node-utils/blob/master/src/coffee/helpers/repeater.coffee#L90-L96

Provide through IT as an example, not function of sync..